lstchain/mc/sensitivity.py

```diff
@@ -351,7 +352,11 @@ def find_best_cuts_sensitivity(simtelfile_gammas, simtelfile_protons,
     gammaness_bins, theta2_bins = bin_definition(n_bins_gammaness, n_bins_theta2)
 
     # Extract spectral parameters
-    dFdE, crab_par = crab_hegra(energy)
+    if(shape == "PowerLaw"):
+        dFdE, crab_par = crab_hegra(energy)
+    elif(shape == "LogParabola"):
+        dFdE, crab_par = crab_magic(energy)
+
```

Comment:
General comment: for differential sensitivity calculations, using the HEGRA power-law spectrum, or a more accurate log-parabola parametrization, will make no difference. The effect would just go through the difference in hardness **within each energy bin** , and hence is very small for the usually narrow (5 bins/dex) bins we use.

The log-parabola weighting is instead crucial for comparisons of Crab data with MC gammas, for example to obtain parameter distributions, because in that case the energy range spanned by events entering one such comparison may be arbitrarily wide.